docs/topics/spiders.rst

```diff
@@ -283,7 +283,19 @@ Spider arguments are passed through the :command:`crawl` command using the
 
     scrapy crawl myspider -a category=electronics
 
-Spiders receive arguments in their constructors::
+Spider arguments are exposed as attributes::
```

Comment:
Hm, I think it'll need more than an example.
Looking at the example alone it looks like broken code. :)

Perhaps adding a default as a class attribute, like:

    import scrapy

    class MySpider(scrapy.Spider):
        name = 'myspider'
        category = 'default-category'  # use -a category=another from the cmdline to override this

        def parse(self, response):
            products = response.css('.%s' % self.category)

This may deserve its own section in the docs really. :)